<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentsNavWin</class>
 <widget class="QWidget" name="ContentsNavWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnToTestForm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(100, 178, 255);
background-color: rgba(82, 84, 84, 179);
border:2px solid rgb(0, 178, 255);</string>
         </property>
         <property name="text">
          <string>To Test Form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToMVForm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(100, 178, 255);
background-color: rgba(82, 84, 84, 179);
border:2px solid rgb(0, 178, 255);</string>
         </property>
         <property name="text">
          <string>To Machine Vision Form</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToNewControlForm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(100, 178, 255);
background-color: rgba(82, 84, 84, 179);
border:2px solid rgb(0, 178, 255);</string>
         </property>
         <property name="text">
          <string>To New Control Form</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbT">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbO">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbE">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
